AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -159,6 +159,7 @@ www.kinopoisk.ru##div[class^="trailer-promo"]
 !
 ! SECTION: Self-promo - Regular rules
 !
+androidadult.com##.support.box
```

Comment:
Usually the rules for blocking annoyances with donations are added to widgets:
https://github.com/AdguardTeam/AdguardFilters/blob/f2d53e8cf875e79c7912087a455a61b20aa73314/AnnoyancesFilter/Widgets/sections/widgets.txt#L2378